docs/changelog.txt

```diff
@@ -1,3 +1,37 @@
+0.5.0 (2020-04-26)
+------------------------------------------------------------------------
+- Feature: Support for RoadTypes (a.k.a NRT)
+- Feature: Industries now support up to 16 accepted cargos and 16 produced cargos
```

Comment:
Looking at the commit, this isn't nearly enough information or even an accurate summary.

It doesn't only *add* support for more cargos, it removes the previous syntax and properties in a way that will break all industry-related grfs.

The changelog needs to say that, and either list the required changes or link to somewhere that does.

(IMO such a big change should really have had a deprecation period, but too late now),

Reply:
> (IMO such a big change should really have had a deprecation period, but too late now),

I initially left support for the old syntax in, but then was told that the NML way is to remove all traces of the old way of doing things when a new and better way is introduced.

Agree that the changelog should at least warn that the syntax has changed and all projects that define industries need to be updated, or stay on the old version.